snaps/features/custom-ui/with-jsx.md

```diff
@@ -861,6 +861,7 @@ module.exports.onHomePage = async () => {
 <img src={require("../../assets/custom-ui-heading.png").default} alt="Text UI example" width="450px" style={{border: "1px solid #DCDCDC"}} />
 </p>
 
+<<<<<<< HEAD
```

Comment:
This is a hanging merge conflict line right?

Reply:
yes, it's since been fixed :-)